support-frontend/assets/components/checkoutNudge/checkoutNudge.tsx

```diff
@@ -0,0 +1,150 @@
+		background-repeat: no-repeat;
+		background-position: right;
+		background-image: url(https://media.guim.co.uk/2d33e52f89462481b77f0fd419d62a55fb70c0f0/0_0_274_202/274.png);
+		${from.mobileMedium} {
+			background-image: url(https://media.guim.co.uk/91d324df80b882d314dcd35f23dcffea8e346824/0_0_329_188/329.png);
+		}
+		${from.mobileLandscape} {
+			background-image: url(https://media.guim.co.uk/2fcda5a622b598515997c0ce0ff98faffec3826d/0_0_415_188/415.png);
+		}
+		${from.tablet} {
+			margin-top: ${space[2]}px;
+			background-image: url(https://media.guim.co.uk/933daaf47129bc1ed1f9af4171d3d5a637fadf2c/0_0_419_213/419.png);
+		}
+		${from.desktop} {
+			background-image: url(https://media.guim.co.uk/29edb4f891687e2188cc44f352372536ee41a04c/0_0_493_190/493.png);
+		}
+	`,
+	top: css`
+		margin-top: ${space[2]}px;
+		margin-bottom: -${space[1]}px;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		${from.mobileMedium} {
+			margin-bottom: -${space[3]}px;
+		}
+	`,
+	topheading: css`
+		margin-top: ${space[3]}px;
+	`,
+	heading: (backColor: string, marginBottom: number) => css`
+		margin-left: 10px;
+		color: ${backColor};
+		${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+		line-height: 108%;
+
+		${from.mobileMedium} {
+			margin-left: 12px;
+			margin-bottom: ${marginBottom}px;
+			${headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+			line-height: 108%;
+		}
+		${from.tablet} {
+			${headline.small({ fontWeight: 'bold', lineHeight: 'tight' })};
+			line-height: 108%;
+		}
+	`,
+
+	para: css`
+		margin-top: ${space[1]}px;
+		margin-left: 10px;
+		margin-right: 10px;
+		margin-bottom: ${space[2]}px;
+		${textSans.small({ lineHeight: 'regular' })};
+		line-height: 130%;
+		${from.mobileMedium} {
+			margin-left: 12px;
+			max-width: 470px;
+		}
+		${from.tablet} {
+			${textSans.medium({ lineHeight: 'regular' })};
+			line-height: 130%;
+		}
+	`,
+	link: css`
+		margin-left: 10px;
+		${textSans.small({ lineHeight: 'regular' })};
+		line-height: 135%;
+		padding-bottom: ${space[5]}px;
+
+		${from.mobileMedium} {
+			margin-left: 12px;
+		}
+		${from.tablet} {
+			${textSans.medium({ lineHeight: 'tight' })};
+			line-height: 108%;
+		}
+	`,
+	alink: css`
+		color: ${brand[500]};
+		text-decoration: underline;
+		&:hover {
+      fontWeight:'bold';
+			cursor: pointer;
+		},
+	`,
+};
+
+export type CheckoutNudgeProps = {
+	contributionType: ContributionType;
+	nudgeDisplay: boolean;
+	nudgeTitleCopySection1: string;
+	nudgeTitleCopySection2: string;
+	nudgeParagraphCopy: string;
+	onNudgeClose: () => void;
+	onNudgeClick: () => void;
+};
+
+export function CheckoutNudge({
+	contributionType,
+	nudgeDisplay,
+	nudgeTitleCopySection1,
+	nudgeTitleCopySection2,
+	nudgeParagraphCopy,
+	onNudgeClose,
+	onNudgeClick,
+}: CheckoutNudgeProps): JSX.Element {
+	const displayValid = contributionType === 'ONE_OFF' && nudgeDisplay;
+	return (
+		<div css={styles.container}>
```

Comment:
Is it common practise to define CSS rules as props within a `styles` object rather than defining them as individual variables? I can see it works, but my concern is ESLint would not warn us if rules within this `styles` object are no longer used, which it would do if they were individual variables? eg. 

```
const styles = { 
  container: css``,
  top: css``
 }
```

VS 

```
const container = css: ``;
const top = css: ``;
```

This isn't a blocker, but it's worth considering/thinking about.

Reply:
Yes makes sense, split them out

Reply:
Completed- yes makes sense for linter to spot, have split them out now